### PR TITLE
Fix typo in conversion/index.js

### PR DESCRIPTION
**Pull request by eltociear**
**Description**

Fixed typo.
```
specificy -> specificity
```

**Checklist**

* [ ] There is a related GitHub issue
* [ ] Tests are included if applicable
* [ ] Any added code is fully documented

**Issue**

Resolves #???

### CODE REVIEW

**Comment by eltociear** [comment]
I have read the CLA Document and I hereby sign the CLA

**Review by CryptoGodfatherVA4 — APPROVED** [review]

**Comment by gantunesr** [comment]
hey @eltociear, I believe there's no typo here, the use of "specify" is the same as to "identify clearly". Let me know what you think

File: app/util/conversion/index.js [hunk 1]
```diff
@@ -163,7 +163,7 @@ const conversionUtil = (
 
 const getBigNumber = (value, base) => {
 	if (!isValidBase(base)) {
-		throw new Error('Must specificy valid base');
+		throw new Error('Must specificity valid base');
```
Comment (sethkfman) [inline app/util/conversion/index.js:166]:
```suggestion
		throw new Error('Must specify valid base');
```

**Review by jpcloureiro — APPROVED** [review]

File: app/util/conversion/index.js [hunk 2]
```diff
@@ -173,9 +173,9 @@ const conversionUtil = (
 };
 
 const getBigNumber = (value, base) => {
-  if (!isValidBase(base)) {
-    throw new Error('Must specificy valid base');
-  }
+	if (!isValidBase(base)) {
```
Comment (sethkfman) [inline app/util/conversion/index.js:176]:
```suggestion
  if (!isValidBase(base)) {
```

File: app/util/conversion/index.js [hunk 3]
```diff
@@ -174,8 +174,8 @@ const conversionUtil = (
 
 const getBigNumber = (value, base) => {
   if (!isValidBase(base)) {
-    throw new Error('Must specificy valid base');
-  }
+		throw new Error('Must specify valid base');
```
Comment (sethkfman) [inline app/util/conversion/index.js:177]:
```suggestion
    throw new Error('Must specify valid base');
```

File: app/util/conversion/index.js [hunk 4]
```diff
@@ -174,8 +174,8 @@ const conversionUtil = (
 
 const getBigNumber = (value, base) => {
   if (!isValidBase(base)) {
-    throw new Error('Must specificy valid base');
-  }
+    throw new Error('Must specify valid base');
+	}
```
Comment (sethkfman) [inline app/util/conversion/index.js:178]:
```suggestion
  }
```

**Pull request merged**